Detections/SlackAudit/SlackAuditMultiple archivedFilesUploadedInShortTimePeriod.yaml

```diff
@@ -0,0 +1,32 @@
+id: 995fada8-9101-4b4b-a75d-e11bcb80e9be
+name: SlackAudit - Multiple archived files uploaded in short time period
+description: |
+  'This query helps to detect fact of uploading multiple archived files in short time period.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SlackAuditAPI
+    dataTypes:
+      - SlackAudit_CL
+queryFrequency: 30m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: Collection
+relevantTechniques:
+  - T1005, T1560
+query: |
+  let lbtime = 30m;
+  let threshold = 10;
+  SlackAudit
+  | where TimeGenerated > ago(lbtime)
+  | where DvcAction =~ 'file_uploaded'
+  | extend FE = extract(@'.*\.(.*)$', 1, EntityFileName)
+  | where FE in~ ('tar', 'bz2', 'gz', 'tgz', 'Z', 'tbz2', 'zst', 'zip', 'zipx', '7z', 'rar', 'sfx')
+  | summarize count() by SrcUserName
```

Comment:
Same as above, may want to bring thru more than just count and SrcUserName for the analyst who receives this alert.  Also, use the bin(TimeGenerated, 30m) here, for example `| summarize count() by bin(TimeGenerated, 30m), SrcUserName`

Reply:
Fixed, Pls check